utils.py

```diff
@@ -246,3 +249,16 @@ def check_software_installed(list_software=['sct']):
             logging.error("'{}' is not installed. Please install it before using this program.".format(software))
             install_ok = False
     return install_ok
+
+
+def create_empty_mask(fname, fname_label):
+    """
+    Create empty mask from reference image
+    :param fname: path to reference image
+    :param fname_label: path to output mask under derivatives
+    """
+    img = nib.load(fname)
+    data = np.zeros(img.shape)
+    img_mask = nib.Nifti1Image(data, img.affine, img.header)
```

Comment:
While it is implied, I find it better to add the argument name also, i.e 
```
img_mask = nib.Nifti1Image(data, affine=img.affine, header=img.header)
```
You don't _have_ to do this but it just makes things more explicit

Reply:
Great suggestion! Done in https://github.com/spinalcordtoolbox/manual-correction/pull/15/commits/b1f341c420f073350ad2bd45184af8a6f6e03785